openwpm/mp_logger.py

```diff
@@ -207,16 +207,6 @@ def _initialize_sentry(self):
         )
         self._event_handler = EventHandler(level=self._log_level_sentry_event)
         sentry_sdk.init(dsn=self._sentry_dsn, before_send=self._sentry_before_send)
-        with sentry_sdk.configure_scope() as scope:
```

Comment:
Because we no longer have easy access to the the storage config (as it's contained in the storage providers) but I think I found a way around it by just using `__str__`  which should be a "nicely printable string representation of an object".